kettle/make_json_test.py

```diff
@@ -634,27 +634,15 @@ class ParseJsonTest(unittest.TestCase):
         ),
         ('No Suite Name',
          'kettle/testdata/junit_no_suite_name.xml',
-         [{'failed': True,
-           'failure_text': 'simple_test.go:186: Running tests in file plumbing',
-           'name': 'unknown TestSimpleFile',
-           'time': 0.0},
-          {'name': 'unknown TestSimpleFile/plumbing/min/min_program', 'time': 0.0},
-          {'name': 'unknown TestSimpleFile/plumbing/eval_results/error_result',
-           'time': 0.0}],
+         [],
         ),
         ('Failure as Message',
          'kettle/testdata/failure_message.xml',
-         [{'failed': True,
-           'failure_text': 'simple_test.go:186: Running tests in file plumbing',
-           'name': 'unknown TestSimpleFile',
-           'time': 0.0}],
+         [],
```

Comment:
I would update testdata to be valid here, give testsuite a name; this test case shouldn't be exercising the no-suite-name case, that's done above

Reply:
I am fine with making it more robust, I opted for fail out because it made it a little difficult to track down failures with an unknown back reference, but within a job it seems there are typically very few tests with the same name across test suites. I will update, thanks.